### PR TITLE
added color theory in visual design

### PR DESCRIPTION
**Pull request by Pallavi024**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> [I have written this article myself though I have taken some reference from youtube videos for the color wheel part  ]

### CODE REVIEW

**Comment by the-ethan-hunt** [comment]
Content completely genuine. :smile: 
@RN0311 will check the article for typos. If present please correct them and I will merge your request.

**Comment by Pallavi024** [comment]
Okay

File: guides/Visual_design/color theory/color theory.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Overview
+
+Color theory is a combination of science and art of color. It explains how humans perceive color, how colors mix, match or clash and various ways in which colors can be classified and used.
+
+# RGB COlOR MODEL
+
+This model uses the primary colors red green and blue in different proportions to create all the necessary colors .When you mix red,green and blue colors to their full extent i.e 100% the resultant color is white and when they are mixed to their lowest degree the resultant color is black. So basically you start with black and as you add lights of various colors you get white. Thus RGB color model is also called as additive color model. Colors are much more vibrant in case of rgb as compared to cmyk.
```
Comment (RN0311) [inline guides/Visual_design/color theory/color theory.md]: 1) Insert a comma between, red and green. 
2) Extra space between colors and full stop.

File: guides/Visual_design/color theory/color theory.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# CMYK COLOR MODEL
+
+ This model uses the colors cyan, magenta, yellow and black in various amounts to create all of the necessary colors. When you mix cyan magenta and yellow to their full extent i.e 100% the resultant color is black thus as you remove or subtract (absorb) various proportions of these colors you get a variety of different colors ,thus CMYK model is also called as subtractive color model .
+
+# HEX CODE 
+
+colors can also be represented using hexadecimal values also called as hex code.The code starts with a hash sign (#) and is followed by six hex values or three hex value pairs for example #AFD645. You can get hex codes for various colors [here]("https://htmlcolorcodes.com/"). Hex codes are generally used for html pages.
```
Comment (RN0311) [inline guides/Visual_design/color theory/color theory.md]: 1) Capital "C" in colors - the beginning of the sentence.

File: guides/Visual_design/color theory/color theory.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# COLOR WHEEL
+
+A color wheel primarily consists of 6 parts. There are three primary colors red,blue and yellow.Colors formed by mixing primary colors are called secondary colors .
+
+- Red   +   Yellow   = Orange (thus orange goes between red and yellow in the color wheel)
+- Blue  +   Yellow   = Green  (thus green goes between blue and yellow in the color wheel)
+- Blue  +   Red      = Purple (thus purple goes between blue and red in the color wheel)
+
+thus each primary color has a secondray color to complement it and as explained above  they  are represented across each other on the color wheel.
```
Comment (RN0311) [inline guides/Visual_design/color theory/color theory.md]: 1) Spacing issues in **above** and **they** and **are**. Kindly check.

**Comment by the-ethan-hunt** [comment]
A few typos still remain, but it's alright! Please take care next time :smiley: 
Merging it.

**Comment by Pallavi024** [comment]
Yeah sure, I'll take care from next time
Thank you 😀

**Pull request merged**